RELEASING_RAILS.md

```diff
@@ -113,8 +113,8 @@ then realize it is broken.
 
 IMPORTANT: The Action Cable client and Action View's UJS adapter are released
 as NPM packages, so you must have Node.js installed, have an NPM account
-(npmjs.com), and be a package owner for `actioncable` and `rails-ujs` (you can
-check this via `npm owner ls actioncable` and `npm owner ls rails-ujs`) in
+(npmjs.com), and be a package owner for `@rails/actioncable` and `@rails/ujs` (you can
```

Comment:
As well as the packages for `activestorage` and `actiontext` now too, right?

Who can the rest of the core team as owner by the way?

Reply:
Yeah, I'll update this list. And, I'm working with Rafael to make sure the core team are all set up with publish permissions.